src/xx-apk

```diff
@@ -10,7 +10,7 @@ if [ -z "$XX_APK_NOLOCK" ]; then
   elif [ ! -d /var/lock ]; then
     mkdir -p /var/lock
   fi
-  cd -
+  cd - >/dev/null
```

Comment:
@tonistiigi figured this out, thanks!

Didn't know `cd` could output :cold_sweat: